python/taichi/lang/ops.py

```diff
@@ -579,3 +579,17 @@ def rescale_index(a, b, I):
         if a.shape[n] < b.shape[n]:
             Ib.entries[n] = I.entries[n] * (b.shape[n] // a.shape[n])
     return Ib
+
+
+def get_addr(f, indices):
+    """Returns the address (for CUDA/x64) for index 'indices' of field 'f'. Currently, this function can only be called inside a kernel.
```

Comment:
super nits: indices and f should be quoted with a single backquote "`"

Reply:
Ah right. will fix.